iphone/Classes/TiApplepayModule.m

```diff
@@ -67,6 +67,8 @@ - (NSNumber *)isSupported:(id)unused
 
 - (NSNumber *)canMakePayments:(id)args
 {
+    args = [args objectAtIndex:0];
```

Comment:
Can you use `ENSURE_SINGLE_ARG(args, NSDictionary);` instead and ping back if it works? 😙

Reply:
Yes it works